README.md

```diff
@@ -1 +1,141 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Introduction
```

Comment:
In the Introduction discuss what the problem is and what you need to do, then go about  explaining your approach to solving the problem. Also put a link to the Trustly problem challenge when describing the task.

Reply:
added link and discussed problem